go/vt/vtctl/grpcvtctlclient/client.go

```diff
@@ -96,3 +96,8 @@ func (client *gRPCVtctlClient) Close() {
 func init() {
 	vtctlclient.RegisterFactory("grpc", gRPCVtctlClientFactory)
 }
+
+// NewClient instantiates a vtctl client instance from address
+func NewClient(address string) (vtctlclient.VtctlClient, error) {
```

Comment:
@deepthi , I need to test vtctl using grpc, but there was no way to get instance of vtctl, that is why I added this public method. Let me know if there is some other to achieve this.

Reply:
Can't you use the `VtctlClientProcess` that you have already added?

Reply:
Yes, I have added test using `VtctlClientProcess` , but since python testcase uses both way of testing i.e. vtctlclient and grpc connection to vtctl, I thought of doing it both way.

Reply:
I don't think that is necessary. Some of what the python test is doing is probably to test the python grpc client itself.

Reply:
I have removed that in last commit, please review again.